src/nvim/options.lua

```diff
@@ -3401,6 +3401,13 @@ return {
         apply equally to 'grepprg'.
         This option cannot be set from a |modeline| or in the |sandbox|, for
         security reasons.
+        If ripgrep is available (|executable()|) then this option and |'grepformat'|
+        default to "rg --vimgrep ..." and "%f:%l:%c:%m" respectively.
+        Ripgrep is a faster, recursive grep that respects gitignore rules
+        and skips hidden and binary files by default, and supports most of the common grep options.
```

Comment:
No need to explain what `ripgrep` is -- but the different defaults (`.gitignore`) should be called out (and possibly disabled by default; some people -- myself included -- prefer the `grep` default).